.vscode/react-native.code-snippets

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
kinda random, but what is this file?

Reply:
its a [code snippet](https://code.visualstudio.com/docs/editor/userdefinedsnippets) file! enter `FC` in an empty file `.tsx` file on this branch to test what it does 🙂

Reply:
oh shoot that's incredible! I just peeked the documentation, that's pretty cool! I couldn't get it to work even after turning on `"editor.tabCompletion": "on"` but I'm not too worried about getting this working atm, I'll figure it out eventually. Good stuff!!